apps/guide/src/content/03-topics/05-intents.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Intents
+category: Topics
+---
+
+# Intents
+
+Intents are an important part of establishing a WebSocket connection, as they define behaviour regarding gateway events and impact received data via the REST API.
+
+## Usage
+
+```js
+import { Client, GatewayIntentBits } from 'discord.js';
+
+const client = new Client({
+	intents: [GatewayIntentBits.Guilds],
+});
+```
+
+This is the most basic usage of intents for discord.js. By specifying _`GatewayIntentBits.Guilds`_, we will receive gateway events regarding guilds. This includes receiving information about guilds as the bot starts up, such as the roles in the guild and the respective shard id.
+
+The full list of _`GatewayIntentBits`_ one can specify can be found <DiscordAPITypesLink type="enum" parent="GatewayIntentBits">on the documentation</DiscordAPITypesLink>. An explanation of what each intent does can be found [on Discord's documentation](https://discord.com/developers/docs/topics/gateway#list-of-intents).
+
+## Considerations
+
+In discord.js, some intents require an extra bit of consideration.
+
+### _`GatewayIntentBits.Guilds`_
+
+discord.js relies heavily on caching in the library. It is recommended to set at least the _`GatewayIntentBits.Guilds`_ intent for discord.js to avoid these pitfalls.
```

Comment:
```suggestion
discord.js relies heavily on caching in the library. We recommend you set at least the _`GatewayIntentBits.Guilds`_ intent to avoid these pitfalls.
```